mappings/net/minecraft/loot/provider/number/BinomialLootNumberProvider.mapping

```diff
@@ -1,6 +1,9 @@
 CLASS net/minecraft/unmapped/C_ysseojpb net/minecraft/loot/provider/number/BinomialLootNumberProvider
 	FIELD f_jlaivpdi n Lnet/minecraft/unmapped/C_iajmfyig;
 	FIELD f_lbshxsdz p Lnet/minecraft/unmapped/C_iajmfyig;
+	METHOD <init> (Lnet/minecraft/unmapped/C_iajmfyig;Lnet/minecraft/unmapped/C_iajmfyig;)V
+		ARG 1 n
+		ARG 2 p
 	METHOD m_ghlbnwvl create (IF)Lnet/minecraft/unmapped/C_ysseojpb;
 		ARG 0 n
 		ARG 1 p
```

Comment:
Would it be possible to maybe get some better args for these fields? Ok if not

Reply:
This is what they eventually get turned into in the code itself, I can't remember exactly what the context is but these names do actually make sense